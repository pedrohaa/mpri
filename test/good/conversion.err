File "good/conversion.f", line 15, characters 7-8:
Type mismatch.
Expected: b -> b
Inferred: a
Hypotheses:
_ = a
__1_ = b -> b
