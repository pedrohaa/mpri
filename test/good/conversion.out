forall a b. arrow (eq a (arrow b b)) (arrow a (arrow b b))
